packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -570,7 +570,9 @@ const handleOriginResponse = async ({
         "cache-control": [
           {
             key: "Cache-Control",
-            value: "public, max-age=0, s-maxage=2678400, must-revalidate"
+            value: hasFallback.fallback
+              ? "no-store" // fallback should never be cached
```

Comment:
possibly not the right cache-control header, maybe `max-age=0, must-revalidate` is better?